src/psyclone/transformations.py

```diff
@@ -324,13 +341,35 @@ def apply(self, node1, node2):
 
 
 class DynamoLoopFuseTrans(LoopFuseTrans):
-    '''Performs error checking before calling the
-        :py:meth:`~LoopFuseTrans.apply` method of the
-        :py:class:`base class <LoopFuseTrans>` in order
-        to fuse two Dynamo loops.
+    ''' Dynamo0.3 API application of the
+    :py:class:`base class <LoopFuseTrans>` in order to fuse two Dynamo
+    Loops after performing validity checks. For example:
 
+    >>> from psyclone.parse.algorithm import parse
+    >>> from psyclone.psyGen import PSyFactory
+    >>>
+    >>> API = "dynamo0.3"
+    >>> FILENAME = "alg.x90"
+    >>> ast, invokeInfo = parse(FILENAME, api=API)
+    >>> psy = PSyFactory(API, distributed_memory=False).create(invoke_info)
+    >>> schedule = psy.invokes.get('invoke_0').schedule
+    >>>
+    >>> from psyclone.transformations import DynamoLoopFuseTrans
+    >>> ftrans =  DynamoExtractRegionTrans()
+    >>>
+    >>> new_schedule, memento = ftrans.apply(schedule.children[0],
```

Comment:
I think this is still the same as it wasn't changes in #454 or #400 (see the comment above).

Reply:
Modified as requested and updated transformation scripts in `examples/dynamo/eg6` to reflect this way of applying the transformation. Also corrected
```
ftrans =  DynamoExtractRegionTrans()
```
to
```
ftrans =  DynamoLoopFuseTrans()
```

Reply:
Thanks for fixing. You made a small mistake (```children[1]``` instead of ```schedule[1]```) but I've just updated that myself rather than request another review.